src/lj_jitlog.c

```diff
@@ -181,6 +181,26 @@ static void memorize_proto(JITLogState *context, GCproto *pt)
   log_gcproto(context->g, pt, proto_bc(pt), proto_bc(pt), mref(pt->k, GCRef),  lineinfo, linesize, proto_varinfo(pt), (uint32_t)vinfosz);
 }
 
+static void memorize_func(JITLogState *context, GCfunc *fn)
+{
+  lua_State *L = mainthread(context->g);
+  int i;
```

Comment:
Why define `int` here and not in the `for` loop?

Reply:
I had some memory of getting warnings from gcc based on the default c language version but tried just now with -pedantic and -Wall and got nothing. The rest of the file is already the same style